Clarify point in time restore geo-redundant backup (#8589)
MicrosoftDocs/sql-docs: azure-sql/database/recovery-using-backups.md

You generally restore a database to an earlier point for recovery purposes. You can treat the restored database as a replacement for the original database or use it as a data source to update the original database.

> [!IMPORTANT]

> > - You can perform point-in-time restore on the same server. Cross-server and Cross subscription Point-in-time restore is not supported yet. Refer to the [Geo-restore](#Geo-Restore) section to restore to a different region for databases set with geo-redundant backup storage.

Reviewer: ```suggestion
> - You can perform point-in-time restore on the same server. Cross-server and cross subscription point-in-time restore is not supported. For more information about restoring databases with geo-redundant backup storage to a different region, see the [Geo-restore](#geo-restore) section.
```
